recipes/gcc/all/conandata.yml

```diff
@@ -1,4 +1,7 @@
 sources:
+  "12.2.0":
+    sha256: ac6b317eb4d25444d87cf29c0d141dedc1323a1833ec9995211b13e1a851261c
+    url: https://ftp.gnu.org/gnu/gcc/gcc-12.2.0/gcc-12.2.0.tar.gz
```

Comment:
is it worth sneaking in 11? https://github.com/conan-io/conan-center-index/pull/12165/files#diff-2c06bb76f54ae09bfa61f45e8e134b61459bfd1b3566831b2c005aaf5d7546faR5-R7

Reply:
Happy to do that if you'd prefer - I'd excluded it because I haven't been able to test it locally due to the same glibc issue I've had with 10.2.0.